release_notes/48-release-notes.adoc

```diff
@@ -395,5 +396,27 @@ This release of {product-title-short} 4.8 provides the following bug fixes:
 * Before this update, excessive logging of telemetry collection status resulted in an abundance of log entries. This update configures telemetry collection to not emit repeated logs continuously, resolving the issue and significantly reducing log volume.
 //ROX-29613 - Olivier
 * Before this update, Central sometimes stored external IPs in the database even after users deleted their corresponding deployments. This resulted in inaccessible and stale data, leading to a loss of storage and memory and potential memory exhaustion. With this release, Central no longer stores external IP information for deleted deployments, resolving these issues.
+//ROX-30366
+* Before this update, the upgrade to Golang gRPC 1.67 and later caused problems with gRPC connections that affected multiple users. This issue prevented gRPC connections and blocked communications between Central and Sensor. With this release, the `GRPC_ENFORCE_ALPN_ENABLED` flag has been added in {product-title-short}. The default value disables the Application-Layer Protocol Negotiation (ALPN) enforcement, and therefore allows the connection between Sensor and Central as well as the communication between the components.
+
+[id="about-this-release-482_{context}"]
+== About release 4.8.2
+
+*Release date* {{ga-date-482}
+
+This release of {product-title-short} 4.8 provides the following bug fixes:
+
+//ROX-30339 - does not need to be included
+//ROX-30307 & ROX-30289
+* With this release, the initialization of image rankers is moved from the critical startup path. Additionally, the query pattern was improved to stop retrieving excessive data, thereby improving startup time and reducing memory consumption.
```

Comment:
Nit: Remove "With this release" and start with "The initialization ..." - maybe redundant given line 407.